packages/node_modules/pouchdb-utils/src/uuid.js

```diff
@@ -39,7 +41,17 @@ var chars = (
   'abcdefghijklmnopqrstuvwxyz'
 ).split('');
 function getValue(radix) {
-  return 0 | Math.random() * radix;
+  // Using maxRandom of 255 will produce an un-even spread of random numbers.
+  // Instead, find the biggest number (within 8-bit range) that is a multiple of
+  // radix.
+  var maxRandom = 255 - (256 % radix);
+
+  var randomValue;
+  do {
+    randomValue = randomUInt8();
+  } while (randomValue > maxRandom);
+
+  return randomValue % radix;
```

Comment:
This makes me nervous performance-wise, since we're spinning in a while loop waiting for a random value to exceed some threshold. Is there a simpler way we can use WebCrypto to generate a UUID? Note that UUIDs in PouchDB don't need to be valid uuid.v4() IDs or whatever; just as long as they're strings and not unreasonably long.